self-core-impl/src/main/java/com/selfxdsd/core/GitlabIssues.java

```diff
@@ -37,7 +38,11 @@
  * @author criske
  * @version $Id$
  * @since 0.0.38
- * @todo #728:60min Implement and test method `getById()` for
+ * @todo #774:60min Implement and test method `received()` for
+ *  GitlabIssues by following GithubIssues as model.
+ * @todo #775:60min Implement and test method `open()` for
+ *  GitlabIssues by following GithubIssues as model.
+ * @todo #776:60min Implement and test method `search()` for
```

Comment:
@criske the ticket number in a ``todo`` should always be the number of the originating ticket, not the number which the resulting Issue will have. 

In this case, all 3 todos should have ``#728``, since this is the originating ticket, the one which you are solving right now.